### PR TITLE
Run upgrade on specific Satellite and Capsule

### LINKED ISSUE #274: Create Upgrade Automation Product User friendly.
Currently the Upgrade Automation is solely used for Testing purpose.
So make it in such way that Product User can use it to upgrade there Satellite.

### PR DESCRIPTION
**Pull request by jyejare**
This will get 'SATELLITE' and 'CAPSULE' hostnames from user as Environment Variables and will run upgrade on them.

closes #274

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1895,7 +1899,8 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     capsule_id = str(
         run(initials + 'capsule list | grep {0}'.format(
             env.get('capsule_host')))).split('|')[0].strip()
-    run(initials+'capsule content synchronize --id {0}'.format(capsule_id))
+    if not os.environ.get('CAP_SUB_DETAILS'):
+        run(initials+'capsule content synchronize --id {0}'.format(capsule_id))
```
Comment (elyezer) [inline automation_tools/__init__.py]: Add space around the `+`

**Comment by elyezer** [comment]
ACK, pending not blocker nitpick.

**Comment by elyezer** [comment]
ACK

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -1852,9 +1852,9 @@ def copy_ssh_key(from_host, to_host):
 
 
 def sync_capsule_tools_repos_to_upgrade(admin_password=None):
-    """This syncs capsule and sat-tools repos in Satellite server.
+    """This syncs capsule repos in Satellite server.
 
-    Useful for upgrading Capsule and Client in feature.
+    Useful for upgrading Capsule in feature.
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: why is client stuff being dropped here? Capsule does require sat-tools repo too.

Comment (jyejare) [inline automation_tools/__init__.py]: 1. Client Upgrade is not ready yet. So temporarily dropped.
2. Capsule doesn't require sat-tools.

Comment (kbidarkar) [inline automation_tools/__init__.py]: NACK, capsule requires sat-tools.

**Comment by kbidarkar** [comment]
NACK

File: automation_tools/__init__.py (hunk 2 above)
Comment (jyejare) [inline automation_tools/__init__.py]: No. Capsule doesn;t requires sat-tools for setup as well as for upgrade.
Refer Point 4 from procedure Procedure 7.1 form Document : https://access.redhat.com/documentation/en-US/Red_Hat_Satellite/6.1/html/Installation_Guide/sect-Red_Hat_Satellite-Installation_Guide-Obtaining_the_Required_Packages_for_the_Capsule_Server.html

**Pull request merged**